Documentation/ceph-nfs-crd.md

```diff
@@ -4,19 +4,22 @@ weight: 3100
 indent: true
 ---
 
-# Ceph NFS Gateway CRD
+# Ceph NFS Server CRD
 
 ## Overview
+Rook allows exporting NFS shares of the filesystem or object store through the CephNFS custom
+resource definition. This will spin up a cluster of
+[NFS Ganesha](https://github.com/nfs-ganesha/nfs-ganesha) servers that coordinate with one another
+via shared RADOS objects. The servers will be configured for NFSv4.1+ access only, as serving
+earlier protocols can inhibit responsiveness after a server restart.
 
-Rook allows exporting NFS shares of the filesystem or object store through the CephNFS custom resource definition. This will spin up a cluster of [NFS Ganesha](https://github.com/nfs-ganesha/nfs-ganesha) servers that coordinate with one another via shared RADOS objects. The servers will be configured for NFSv4.1+ access, as serving earlier protocols can inhibit responsiveness after a server restart.
+> **WARNING**: We do not recommend using NFS in Ceph v16.2.0 through v16.2.6. If you are using Ceph
```

Comment:
If we recommend this, should we enforce this in our code for newer deployments?

Reply:
We talked in huddle about not making this a hard rule. Users can deal with the hiccups if they use a non-recommended version.